src/index.css

```diff
@@ -11,3 +13,72 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+:root {
+  --section-x-padding: 80px 0;
+  --mobile-y-padding: 0 24px;
+
+  --color-black: #000;
+  --color-white: #fff;
+  --color-pink: #FBEAE4;
+
+  --font-main: 'Poppins';
+}
+
+*,
+body,
+div {
+  font-family: var(--font-main);
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
```

Comment:
I did not know you could use both CSS file and styled components (thought it was either way) so this was an eye opener haha. Thanks to you I will work combined in the future :)